index.html

```diff
@@ -107,26 +105,26 @@ <h4 class="primary">Clone the example from <a href="https://github.com/experimen
           </p>
         </li>
       </ol><!-- /.tutorial__steps -->
-
-      <div class="order-hardware">
-        <h2 class="headline-content"><span>Turn it into the real thing</span></h2>
-        <img class="order-hardware__illustration" src="img/order-hardware.jpg" alt="order hardware">
-        <p>If you'd like to try the example on real hardware order the modules via the links below.</p>
-        <a class="button-interaction" href="http://www.co2meter.com/collections/co2-sensors/products/s8-miniature-co2-sensor" target="_blank">Order the CO₂ Sensor
-          <span class="order-hardware__store">co2meter.com</span>
-        </a>
-        <a class="button-interaction" href="http://www.amazon.com/dp/B014H2P4KW/ref=sr_ph?ie=UTF8&amp;qid=1452304434&amp;sr=1&amp;keywords=philips+hue+ambiance+white+and+color" target="_blank">Order the Smartlight
-          <span class="order-hardware__store">amazon.com</span>
-        </a>
-        <a class="button-interaction" href="http://www.newegg.com/Product/Product.aspx?Item=N82E16856501013" target="_blank">Order the Mini PC
-          <span class="order-hardware__store">newegg.com</span>
-        </a>
-        <p>
-          For more information on how to setup the CO₂ Sensor check out this <a href="http://experimental-platform.tumblr.com/private/137559322715/tumblr_o15prdeSZ11ucy30s" target="_blank">little tutorial</a>.
-        </p>
-      </div>
     </div><!-- /.tutorial -->
 
+    <div class="order-hardware section-content">
+      <h2 class="headline-content"><span>Turn it into the real thing</span></h2>
+      <img class="order-hardware__illustration" src="img/order-hardware.jpg" alt="order hardware">
+      <p>If you'd like to try the example on real hardware order the modules via the links below.</p>
+      <a class="button-interaction" href="http://www.co2meter.com/collections/co2-sensors/products/s8-miniature-co2-sensor" target="_blank">Order the CO₂ Sensor
+        <span class="order-hardware__store">co2meter.com</span>
+      </a>
+      <a class="button-interaction" href="http://www.amazon.com/dp/B014H2P4KW/ref=sr_ph?ie=UTF8&amp;qid=1452304434&amp;sr=1&amp;keywords=philips+hue+ambiance+white+and+color" target="_blank">Order the Smartlight
+        <span class="order-hardware__store">amazon.com</span>
+      </a>
+      <a class="button-interaction" href="http://www.newegg.com/Product/Product.aspx?Item=N82E16856501013" target="_blank">Order the Mini PC
+        <span class="order-hardware__store">newegg.com</span>
+      </a>
+      <p>
+        For more information on how to setup the CO₂ Sensor check out this <a href="http://experimental-platform.tumblr.com/private/137559322715/tumblr_o15prdeSZ11ucy30s" target="_blank">little tutorial</a>.
+      </p>
+    </div>
```

Comment:
made ``order-hardware`` a ``section-content`` to ensure that all margins between sections are equal